packages/linter/src/generators/workspace-rules-project/workspace-rules-project.ts

```diff
@@ -16,8 +16,11 @@ import {
 import { getRelativePathToRootTsConfig } from '@nx/js';
 import { join } from 'path';
 import { workspaceLintPluginDir } from '../../utils/workspace-lint-rules';
-import { swcCoreVersion, swcNodeVersion } from 'nx/src/utils/versions';
-import { nxVersion } from '../../utils/versions';
+import {
+  nxVersion,
+  swcCoreVersion,
+  swcNodeVersion,
```

Comment:
I don't think we should import the versions from `@nx/js`, I think we should import a function like `{ addSwcDependencies } from '@nx/js/src/utils/add-swc';`

Reply:
This got lost on rebase; I'll push those changes again.